src/Symfony/Component/HttpKernel/Config/FileLocator.php

```diff
@@ -45,9 +55,28 @@ public function __construct(KernelInterface $kernel, string $path = null, array
     public function locate($file, $currentPath = null, $first = true)
     {
         if (isset($file[0]) && '@' === $file[0]) {
-            return $this->kernel->locateResource($file, $this->path, $first);
+            return $this->kernel->locateResource($file, $this->path, $first, false);
+        }
+
+        $locations = parent::locate($file, $currentPath, $first);
+
+        if (isset($file[0]) && '/' !== $file[0] && '\\' !== $file[0]) {
+            // no need to trigger deprecations when the loaded file is given as absolute path
```

Comment:
an absolute path on Windows will not have `\` as the first char (unless on network mounts), so the detection is broken. See `isAbsolutePath` in the parent class.

Reply:
I've taken over the whole implementation from there